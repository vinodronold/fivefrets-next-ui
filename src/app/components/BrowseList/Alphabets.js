import React from 'react'
import Link from 'next/link'
import { browse } from '../../constants/routes'
import Button from '../Button'
import Box from '../Box'

export default () => (
  <Box>
    {['0 - 1', ...'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('')].map(a => (
      <Link prefetch key={a} href={browse.href(a.substr(0, 1))} as={browse.as(a.substr(0, 1))}>
        <Button style={{ padding: '.5rem' }}>{a}</Button>
      </Link>
    ))}
  </Box>
)
