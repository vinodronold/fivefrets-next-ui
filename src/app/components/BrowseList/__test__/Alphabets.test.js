import React from 'react'
import renderer from 'react-test-renderer'
import Alphabets from '../Alphabets'

jest.mock('next/router')

describe('components/BrowseList/Alphabets', () => {
    it('should match the snapshot', () => {
        const _alphabet = renderer.create(<Alphabets />)
        expect(_alphabet).toMatchSnapshot()
    })
})