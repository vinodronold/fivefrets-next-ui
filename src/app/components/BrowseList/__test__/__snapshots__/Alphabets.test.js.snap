// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/BrowseList/Alphabets should match the snapshot 1`] = `
<div
  className="jsx-3706451951 "
>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    0 - 1
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    A
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    B
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    C
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    D
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    E
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    F
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    G
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    H
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    I
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    J
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    K
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    L
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    M
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    N
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    O
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    P
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    Q
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    R
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    S
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    T
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    U
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    V
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    W
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    X
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    Y
  </button>
  <button
    className="jsx-856601377 jsx-3822723886 "
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "padding": ".5rem",
      }
    }
  >
    Z
  </button>
</div>
`;
